src/14_legendre.jl

```diff
@@ -0,0 +1,418 @@
+@doc raw"""
```

Comment:
why `@doc raw`

Reply:
using <code>@doc raw</code> you don't have to escape all the special chars for latex rendering.
see [Documenter.jl's documentation](https://juliadocs.github.io/Documenter.jl/stable/man/latex/index.html)
> To avoid needing to escape the special characters in docstrings the raw"" string macro can be used, combined with @doc: